library/nmstate_interface.py

```diff
@@ -52,8 +52,37 @@
 class AnsibleNMStateInterface(AnsibleNMState):
     def run(self):
         name = self.params["name"]
-        interface_state = get_interface_state(self.interfaces, name)
-        interface_state["state"] = self.params["state"]
+        full_interface_state = get_interface_state(self.interfaces, name)
+
+        interface_state = {}
```

Comment:
You could use here dict comprehension.

Reply:
What excactly are you suggesting? I do not see the use-case for a dict comprehension here.